sample_scripts/third_party_platform_exporting/magento/v1.9/soap_api/README.md

```diff
@@ -0,0 +1,89 @@
+# Magento v1.x Export Scripts
+
+These are some sample scripts that demonstrate how objects can be exported from Magento.
+
+These command line tools look for specific environment variables that describe the Magento system being exported from. Next, they use the SOAP API to retrieve data and display JSON formatted data to the console/standard out.
```

Comment:
I think this section is a bit awkward. What about:

>These command line tools look for specific environment variables that describe the Magento system being exported from. Using that information, they use Magento's SOAP API to retrieve data from the store and output it in JSON format.

>These scripts output data in a format that the Shopify Transporter Tool can consume. The Shopify Transporter Tool can then transform that data into a CSV that can be uploaded to the [Shopify Transporter App](https://apps.shopify.com/transporter).

If that's not good enough, we should involve an actual content person, IMO.